metadata-transformer/src/main/java/no/unit/nva/metadata/NoLowercaseDCPrefixInMetadataFix.java

```diff
@@ -0,0 +1,25 @@
+package no.unit.nva.metadata;
+
+import org.apache.any23.validator.*;
```

Comment:
I know this is your convention, and don't worry, I am, this was just a quick draft to get some feedback on the general idea.

I'm guessing you meant to say that it loads the whole package, but there's really no reason to worry. The compiler replaces the wildcard import with the specific classes used in the code. I believe the real rationale for not using wildcard imports is to avoid namespace conflicts and for developers to be able to read up top which classes are imported, but the first issue can be addressed and the second one is subjective; in fact Clean Code recommends using wildcard imports.